networkd/network_server.ml

```diff
@@ -844,6 +844,35 @@ module Bridge = struct
 				Sysfs.bridge_to_interfaces name
 		) ()
 
+	let get_physical_interfaces _ dbg ~name =
+		Debug.with_thread_associated dbg (fun () ->
+			match !backend_kind with
+			| Openvswitch ->
+				let ifaces = List.filter (fun iface ->
```

Comment:
You can avoid some duplication by first figuring out the real bridge with
```
let real_bridge =
	match Ovs.bridge_to_vlan bridge with
	| Some (parent, _) -> parent
	| None -> bridge
in
```
and then just once
```
Ovs.bridge_to_interfaces real_bridge
|> List.filter (Sysfs.is_physical)
```
Even better, but a helper function to get the "real" bridge in network_utils (module Ovs), because it is also used elsewhere.